src/rpc/client/ts/tests/fixtures.ts

```diff
@@ -74,6 +76,7 @@ export async function mochaGlobalSetup(): Promise<number | undefined> {
   await mochaGlobalTeardown()
 
   const pid = await startServer(rootPath, getClientVersion(), testPort)
+  console.log(pid)
   if (pid) {
```

Comment:
We don't need this line nor creation of the PID file since the server will do this for us now.